### PR TITLE
KOGITO-3073 Kafka persistence example

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-runtimes/pull/825
- https://github.com/kiegroup/kogito-examples/pull/393
- https://github.com/kiegroup/kie-docs/pull/3178

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
@danielezonca @tiagodolphine @MarianMacik please also take a look here

**Review by danielezonca — APPROVED** [review]
Well done! Approving 👍

File: process-kafka-persistence-quarkus/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+</head>
+<body>
+
+<div class="banner lead">
+    Your new Cloud-Native application is ready!
+</div>
+
+<div class="container">
+    <div class="left-column">
+        <p class="lead"> Congratulations, you have created a new Quarkus application.</p>
```
Comment (tiagodolphine) [inline process-kafka-persistence-quarkus/src/main/resources/META-INF/resources/index.html:108]: I think this file can be removed.

**Review by tiagodolphine — APPROVED** [review]

**Pull request merged**